### PR TITLE
ENYO-5415: Fix to call stopPropagation() properly

### PR DESCRIPTION
**Pull request by ybsung**
Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

We called native event's stopPropagation function incorrectly. So we could not stop to propagate a react synthetic event.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

To stop to propagate a react synthetic event, we call react event's stopPropagation function.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

`ev.preventDefault();` was useless. So I removed it.

### Links
[//]: # (Related issues, references)

ENYO-5415

### Comments

Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### CODE REVIEW

**Comment by 0x64** [comment]
LGTM

**Comment by MikyungKim** [comment]
LGTM, but please resolve the conflict.

**Comment by ybsung** [comment]
@MikyungKim 
I fixed the conflict.

File: packages/moonstone/CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
-- `moonstone/ProgressBar.ProgressBarTooltip` not to display unknown props warning
+- `moonstone/Scroller`, `moonstone/VirtualList.VirtualGridList`, and `moonstone/VirtualList.VirtualList` page controls to stop propagating an event if the event is handled
+- `moonstone/ProgressBar.ProgressBarTooltip` not to display unknown props warn
```
Comment (MikyungKim) [inline packages/moonstone/CHANGELOG.md]: Some of the characters are missing.

**Pull request merged**